packages/admin/admin-rte/src/core/translation/stateToHtml.spec.ts

```diff
@@ -0,0 +1,511 @@
+            { key: "aje6k", text: "Headline 5", type: "header-five", depth: 0, inlineStyleRanges: [], entityRanges: [], data: {} },
+            { key: "7u6on", text: "Headline 6", type: "header-six", depth: 0, inlineStyleRanges: [], entityRanges: [], data: {} },
+        ];
+
+        const rawContent = {
+            entityMap: {},
+            blocks,
+        } as RawDraftContentState;
+
+        const content = convertFromRaw(rawContent);
+        const editorState = EditorState.createWithContent(content);
+
+        const { html } = stateToHtml({
+            editorState,
+            options,
+        });
+
+        const expectedHtml = `<p>Normal Text</p>\n<p><strong>Bold Text</strong></p>\n<p><em>Italic Text</em></p>\n<p><em><strong>Bold Italic Text</strong></em></p>\n<p><del>Strikethrough Text</del></p>\n<p>A <sub>Subscript Text</sub></p>\n<p>B <sup>Superscript Text</sup></p>\n<h1>Headline 1</h1>\n<h2>Headline 2</h2>\n<h3>Headline 3</h3>\n<h4>Headline 4</h4>\n<h5>Headline 5</h5>\n<h6>Headline 6</h6>`;
+
+        expect(html).toEqual(expectedHtml);
+    });
+
+    it("should convert the rte editor state with formating into html while keeping the format via tags - formats part 2 (unordered list)", () => {
+        const blocks = [
+            { key: "a9t3", text: "Unordered List", type: "unordered-list-item", depth: 0, inlineStyleRanges: [], entityRanges: [], data: {} },
+            {
+                key: "f4o2c",
+                text: "123456",
+                type: "unordered-list-item",
+                depth: 1,
+                inlineStyleRanges: [{ offset: 3, length: 3, style: "SUB" }],
+                entityRanges: [],
+                data: {},
+            },
+            {
+                key: "7v61p",
+                text: "234",
+                type: "unordered-list-item",
+                depth: 2,
+                inlineStyleRanges: [{ offset: 0, length: 3, style: "ITALIC" }],
+                entityRanges: [],
+                data: {},
+            },
+            {
+                key: "1duir",
+                text: "345",
+                type: "unordered-list-item",
+                depth: 2,
+                inlineStyleRanges: [{ offset: 0, length: 3, style: "BOLD" }],
+                entityRanges: [],
+                data: {},
+            },
+        ];
+
+        const rawContent = {
+            entityMap: {},
+            blocks,
+        } as RawDraftContentState;
+
+        const content = convertFromRaw(rawContent);
+        const editorState = EditorState.createWithContent(content);
+
+        const { html } = stateToHtml({
+            editorState,
+            options,
+        });
+
+        const expectedHtml = `<ul>\n  <li>Unordered List\n    <ul>\n      <li>123<sub>456</sub>\n        <ul>\n          <li><em>234</em></li>\n          <li><strong>345</strong></li>\n        </ul>\n      </li>\n    </ul>\n  </li>\n</ul>`;
```

Comment:
Why does the HTML contain `\n` and whitespaces? Could we remove them?

Reply:
Because of the received html. This would work. `expect(html.replace(/\s|\n/g, "")).toEqual(expectedHtml.replace(/\s|\n/g, ""));`